sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -83,6 +83,7 @@ public partial class AnalyzeHealthcareEntitiesResultCollection : System.Collecti
     public partial class AnalyzeSentimentAction
     {
         public AnalyzeSentimentAction() { }
+        public AnalyzeSentimentAction(Azure.AI.TextAnalytics.AnalyzeSentimentOptions options) { }
```

Comment:
just wondering, do the options configured per action take precedence over the options passed in AnalyzeActionOptions at the client method level?

Reply:
Not really. They are all independent so there is no overriding.
The only property "shared" is `IncludeStatistics` but I am not setting that one if it is passed to the overloads because it doesn't fit in a specific action.
I tried to be very specific about which properties the overload was taking to avoid that confusion.

Is there any other that makes you wonder?

Reply:
Got it - that makes sense.